### ISSUE TITLE
Missing dependency on trailblazer-developer

### ISSUE THREAD

**Issue by samstickland** [trailblazer/trailblazer-operation#56]
After updating trailblazer-operation it no longer has a dependency on trailblazer-developer. This is from my `Gemfile.lock`

```diff
-    trailblazer-operation (0.6.6)
-      trailblazer-activity (>= 0.11.2, < 1.0.0)
-      trailblazer-activity-dsl-linear (>= 0.3.2, < 1.0.0)
-      trailblazer-developer (>= 0.0.8)
+      trailblazer-activity-dsl-linear (>= 0.4.0, < 0.5.0)
+    trailblazer-operation (0.7.2)
+      trailblazer-activity-dsl-linear (>= 0.4.0, < 1.0.0)
```

But it does still try to require it. Without explicitly adding `trailblazer-developer` into my Gemfile this is the stacktrace I get when trying to start my environment:

```
LoadError: cannot load such file -- trailblazer/developer
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/loaded_features_index.rb:89:in `register'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:44:in `require'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/activesupport-6.0.2.2/lib/active_support/dependencies.rb:325:in `block in require'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/activesupport-6.0.2.2/lib/active_support/dependencies.rb:291:in `load_dependency'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/activesupport-6.0.2.2/lib/active_support/dependencies.rb:325:in `require'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/trailblazer-operation-0.7.2/lib/trailblazer/operation/trace.rb:2:in `<main>'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/activesupport-6.0.2.2/lib/active_support/dependencies.rb:325:in `block in require'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/activesupport-6.0.2.2/lib/active_support/dependencies.rb:291:in `load_dependency'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/activesupport-6.0.2.2/lib/active_support/dependencies.rb:325:in `require'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/trailblazer-operation-0.7.2/lib/trailblazer/operation.rb:52:in `<class:Operation>'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/trailblazer-operation-0.7.2/lib/trailblazer/operation.rb:43:in `<module:Trailblazer>'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/trailblazer-operation-0.7.2/lib/trailblazer/operation.rb:8:in `<main>'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/activesupport-6.0.2.2/lib/active_support/dependencies.rb:325:in `block in require'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/activesupport-6.0.2.2/lib/active_support/dependencies.rb:291:in `load_dependency'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/activesupport-6.0.2.2/lib/active_support/dependencies.rb:325:in `require'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/trailblazer-2.1.0/lib/trailblazer.rb:2:in `<main>'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
/Users/samuelstickland/development/hubbado/hubbado_core/config/application.rb:8:in `<top (required)>'
/Users/samuelstickland/development/hubbado/hubbado_core/Rakefile:4:in `require'
/Users/samuelstickland/development/hubbado/hubbado_core/Rakefile:4:in `<top (required)>'
/Users/samuelstickland/.gem/ruby/2.7.2/gems/rake-13.0.3/exe/rake:27:in `<top (required)>'
/Users/samuelstickland/.gem/ruby/2.7.2/bin/bundle:23:in `load'
/Users/samuelstickland/.gem/ruby/2.7.2/bin/bundle:23:in `<main>'
```

**Comment by jamesbebbington** [comment]
Yup, I just came here to file the same issue. Looks to be related to [this `require`](https://github.com/trailblazer/trailblazer-operation/blob/e13b82210a89c89ea03ca18c68ca105e50382c15/lib/trailblazer/operation/trace.rb#L2).

**Comment by yogeshjain999** [comment]
@apotonick This commit introduced it https://github.com/trailblazer/trailblazer-operation/commit/7c30dc6b7593e23179896c2b8fe689d30d8bfae2

Should we extract developer logic and put it into `trb-dev`, by making OP as a dependency ?

**Comment by apotonick** [comment]
Let's just make `developer` a real dependency in `operation` - I guess  99% or our users want `wtf?` anyway.

**Comment by yogeshjain999** [comment]
@samstickland @jamesbebbington released `0.7.3` with the fix, thanks for reporting!
